### ISSUE TITLE
[Issue]: Cannot get image dimensions for logo.svg

### ISSUE THREAD

**Issue by PhantomPhoton** [jellyfin/jellyfin#10026]
### Please describe your bug

During scans, Jellyfin is logging numerous errors about not being able to determine the image dimensions for `logo.svg` files. The files appear to be valid. 

For example, 

[This Logo](https://www.themoviedb.org/t/p/original/oYFsVDtWj8T99qldiALKc4aj7qS.svg) shows up as invalid when scanning, however, when viewed in the browser is valid.

#9581 doesn't appear to completely solve this issue.

### Jellyfin Version

Other

### if other:

10.8.10

### Environment

```markdown
- OS: Ubuntu 22.04
- Linux Kernel: 5.15.0-76-generic
- Virtualization: Docker
- Clients: Browser
- Browser: Chrome 114.0.5735.198
- FFmpeg Version: 5.1.3-Jellyfin
- Playback Method: All methods are impacted
- Hardware Acceleration: None
- GPU Model: None
- Plugins: None
- Reverse Proxy: None
- Base URL: None
- Networking: Host
- Storage: local
```

### Jellyfin logs

```shell
[22:19:40] [ERR] [52] Emby.Server.Implementations.Library.LibraryManager: Cannot get image dimensions for /path/to/logo.svg
System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at SkiaSharp.Extended.Svg.ColorHelper.TryParse(String str, SKColor& color)
   at SkiaSharp.Extended.Svg.SKSvg.ReadPaints(Dictionary`2 style, SKPaint& strokePaint, SKPaint& fillPaint, Boolean isGroup)
   at SkiaSharp.Extended.Svg.SKSvg.ReadElement(XElement e, SKCanvas canvas, SKPaint stroke, SKPaint fill)
   at SkiaSharp.Extended.Svg.SKSvg.ReadElement(XElement e, SKCanvas canvas, SKPaint stroke, SKPaint fill)
   at SkiaSharp.Extended.Svg.SKSvg.LoadElements(IEnumerable`1 elements, SKCanvas canvas, SKPaint stroke, SKPaint fill)
   at SkiaSharp.Extended.Svg.SKSvg.Load(XDocument xdoc)
   at SkiaSharp.Extended.Svg.SKSvg.Load(Stream stream)
   at SkiaSharp.Extended.Svg.SKSvg.Load(String filename)
   at Jellyfin.Drawing.Skia.SkiaEncoder.GetImageSize(String path)
   at Emby.Drawing.ImageProcessor.GetImageDimensions(BaseItem item, ItemImageInfo info)
   at Emby.Server.Implementations.Library.LibraryManager.UpdateImagesAsync(BaseItem item, Boolean forceUpdate)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cvium** [comment]
As far as I know, the fix is not in 10.8

**Comment by cvium** [comment]
Also a duplicate of https://github.com/jellyfin/jellyfin/issues/7626
